.eslintignore

```diff
@@ -0,0 +1 @@
+/themes/*
```

Comment:
```suggestion
/themes/*
/public/*
```
When docs build, they are placed into the "public" folder that can confuse the linter.

Reply:
Done.